ext_mongodb.php

```diff
@@ -7,7 +7,7 @@
 
 interface Serializable extends Type
 {
-	function bsonSerialize() : array;
+	function bsonSerialize() : object;
```

Comment:
I don't think this should be changed. `bsonSerialize()` is allowed to return an array or object to have control over whether classes are serialized as BSON arrays or objects (consider BSONArray and BSONDocument in PHPLIB). Unless you have union types available, this return type hint will need to be changed to `mixed`.